index.html

```diff
@@ -1,12 +1,66 @@
 <!DOCTYPE html>
 <html lang="en">
 <head>
+    <link rel="stylesheet" href="styles/index.css">
     <meta charset="UTF-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Weather Report</title>
 </head>
 <body>
-    
+    <header>
+        <h1> The Weather Report </h1>
+    </header>
+    <main>
+        <section>
+            <h2 id="amazingCity"></h2>
```

Comment:
Remember heading tags (`h1`, `h2`, etc) should be used to title parts of your page, not just for sizing text.

This should have probably been a `p` or `div` instead:

```suggestion
            <p id="amazingCity"></p>
```